### PR TITLE
LPD-30139 Creating liferay-depentencies-zip

### PR DESCRIPTION
**Pull request by pgydev**

### CODE REVIEW

File: release/_package.sh [hunk 1]
```diff
@@ -132,6 +132,23 @@ function package_release {
 
 	lc_cd "${_BUILD_DIR}/release"
 
+	if [[ "${_PRODUCT_VERSION}" == 7.3* ]]
+	then
+		mkdir -p liferay-portal-dependencies-"${_PRODUCT_VERSION}"
```
Comment (natocesarrego) [inline release/_package.sh:137]: @pgydev, shouldn't we name this folder as `liferay-${LIFERAY_RELEASE_PRODUCT_NAME}-dependencies-${_PRODUCT_VERSION}-${_BUILD_TIMESTAMP}"`?

File: release/_package.sh [hunk 2]
```diff
@@ -154,7 +171,7 @@ function package_release {
 
 	lc_cd "${_PROJECTS_DIR}"/liferay-portal-ee
 
-	cp -a sql liferay-${LIFERAY_RELEASE_PRODUCT_NAME}-sql
+	cp -a sql liferay-"${LIFERAY_RELEASE_PRODUCT_NAME}"-sql
```
Comment (natocesarrego) [inline release/_package.sh:174]: @pgydev, can you add this to a separated commit?

The commits should be atomic, e.g. ideally they should do a single action. :slightly_smiling_face:

File: release/_package.sh [hunk 3]
```diff
@@ -132,6 +132,23 @@ function package_release {
 
 	lc_cd "${_BUILD_DIR}/release"
 
+	if [[ "${_PRODUCT_VERSION}" == 7.3* ]]
```
Comment (natocesarrego) [inline release/_package.sh:135]: @pgydev, can you please fix the spelling of the word **dependencies** in your commit message?

Thanks

**Review by natocesarrego — CHANGES_REQUESTED** [review]

File: release/_package.sh [hunk 4]
```diff
@@ -132,6 +132,23 @@ function package_release {
...
+	then
+		mkdir -p liferay-portal-dependencies-"${_PRODUCT_VERSION}"
+
+		cp "${_PROJECTS_DIR}"/liferay-portal-ee/lib/development/hsql.jar liferay-portal-dependencies-"${_PRODUCT_VERSION}"
+
+		cp "${_PROJECTS_DIR}"/liferay-portal-ee/lib/global/*.jar liferay-portal-dependencies-"${_PRODUCT_VERSION}"
+
+		cp "${_PROJECTS_DIR}"/liferay-portal-ee/portal-kernel/portal-kernel.jar liferay-portal-dependencies-"${_PRODUCT_VERSION}"
+
+		find "${_PROJECTS_DIR}"/liferay-portal-ee/tools/sdk/dist -name 'com.liferay.registry.api-*.jar' -exec cp {} liferay-portal-dependencies-"${_PRODUCT_VERSION}" \;
```
Comment (natocesarrego) [inline release/_package.sh:145]: @pgydev, is it covering all these instructions?

File: release/_package.sh (hunk 1 above)
Comment (pgydev) [reply release/_package.sh:137]: @natocesarrego  Yes, you are right we need that, but i think it's enough to put the `${_BUILD_TIMESTAMP}` to the zip file's name.

File: release/_package.sh (hunk 4 above)
Comment (pgydev) [reply release/_package.sh:145]: @natocesarrego  What do you mean by that?

**Pull request closed without merging**